Serialiser_Engine/Objects/BsonSerializers/MethodBaseSerializer.cs

```diff
@@ -88,18 +88,20 @@ public override MethodBase Deserialize(BsonDeserializationContext context, BsonD
             bsonReader.ReadName();
             string methodName = bsonReader.ReadString();
 
-            List<string> paramTypes = new List<string>();
+            List<string> paramTypesJson = new List<string>();
```

Comment:
this improves the clarity of the variable: it's not the name of the type, it is its json representation